playbooks/roles/airship-deploy-tempest/tasks/prepare-tempest.yml

```diff
@@ -0,0 +1,68 @@
+---
+# tasks file for airship-deploy-tempest
+- name: Load generic and openstack mechanism variables
+  include_vars: "{{ item }}"
+  with_items:
+    - "{{ playbook_dir }}/../vars/common-vars.yml"
+    - "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
```

Comment:
should that only be included when the deploy mechanism is openstack?

Reply:
Agree this is useful by 'openstack' deployment mechanism. When I ran with 'kvm' mechanism, I didn't see any side-effect and deployment/ tempest execution ran without error. It may be because we don't have any corresponding kvm specific variables so there is no conflict per se and impact as well.

Reply:
agree that there is no side effect at the moment, but it could be a time tomb and easy to avoid :-).

Reply:
I have made inclusion conditional with latest update. So hopefully saved world from world war 3 :)

Reply:
Requested change was made in earlier patches